prime-router/src/test/kotlin/common/HttpClientUtilsTests.kt

```diff
@@ -0,0 +1,37 @@
+package gov.cdc.prime.router.common
+
+import io.ktor.client.HttpClient
+import junit.framework.TestCase.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import java.lang.reflect.Field
+import kotlin.test.Test
+
+class HttpClientUtilsTests {
+
+    @BeforeEach
+    fun setup() {
+        HttpClientUtils.reset()
+    }
+
+    @Test
+    fun `test default values for client is null`() {
+        // httpClient should be initialized as null
+        val httpClientField: Field = HttpClientUtils::class.java.getDeclaredField("httpClient")
```

Comment:
We should avoid reflection where possible. We treat it as a work around for when we need to do something with a library we don't have source code access to, and should limit it to that use case. What we can do here instead is leverage Spyk (see for usages in code) to assert certain branches of getDefaultHttpClient get called instead of verifying the private `httpClient` directly. Although, we may not need this if we apply the suggested change to default httpClient to an actual object instead of null.